region/maridia/inner-pink.json

```diff
@@ -4672,7 +4724,6 @@
                   "name": "Shinespark",
                   "notable": false,
                   "requires": [
-                    "h_canNavigateUnderwater",
```

Comment:
is this not required?

Reply:
It feels like a grey area to me.  Mainly, you don't need to understand underwater movement for this strat.
You can stand by the door and spark to reach the next node.  There's not really risk of failing if you know how to shinespark.  
But you do pass over an area of danger while underwater, and bad inputs can lead to death.
In this case, to continue, `canNavigateUderwater` will be required anyway.